website/pages/menu.mdx

```diff
@@ -86,6 +86,10 @@ the Menu component. `isOpen` reflects the current open or closed state of the `C
 
 <script>
 export default {
+  data() {
+      return {
+     isOpen: false,
+    },
```

Comment:
I think these lines of code are redundant for now. The scoped slot is sufficient for this.